pipeline/data_store.py

```diff
@@ -62,7 +66,7 @@ class DataStore:
         'measurements'
     ]
 
-    attributes_to_clear = [
+    products_to_clear = [
```

Comment:
If the purpose of this is to save memory, probably we also want to clear detections, measurements, cutouts.  Those aren't as big as the images, but esp. on a very full image, cutouts could be worth cleaning up.  (And if we clean up cutouts, we should clean up the other stuff that goes with them.)